src/main/resources/messages_en_US.properties

```diff
@@ -7,9 +7,8 @@ error.worldIsNull=World of door %DOOR_NAME% does not exist. Removed.
 general.metrics=Metrics enabled. Thank you very much!
 # %PLUGIN_NAME%
 help.title=Help for %PLUGIN_NAME%
-help.setClosed=Set a door as permanent closed. It will only open on player approach.
-help.setTimed=Set a door as timed. It will be open between open and close time. However it will open on player approach if closed. Use a permission or range to avoid this.
-help.setRange=Set the approach range of a door. The range is spherical around the mass center of the door. Default is 10. 0 to disable.
+help.setClosed=Set a door as permanently closed. It will only open on player approach.
+help.setTimed=Set a door as timed. It will be open between opened and closed times. However it will open on player approach if closed. Use a permission or range to avoid this.help.setRange=Set the approach range of a door. The range is spherical around the mass center of the door. Default is 10. 0 to disable.
```

Comment:
You lost a key here ;) 
help.setRange need a new line.

Reply:
my bad